owncloudApp/src/main/java/third_parties/in/srain/cube/GridViewWithHeaderAndFooter.java

```diff
@@ -593,7 +593,7 @@ public boolean hasStableIds() {
         @Override
         public View getView(int position, View convertView, ViewGroup parent) {
             if (DEBUG) {
-                Log.d(LOG_TAG, String.format("getView: %s, reused: %s", position, convertView == null));
+                Log_OC.d(String.format("getView: %s, reused: %s", position, convertView == null));
```

Comment:
This complete logging here is pointless
`public static boolean DEBUG = false;`
It's was **never** switched on. Simply dead code !

Because the title is "_Cleanup project_", I vote for remove this `DEBUG` **and** the logging.

Reply:
You are right, done.